infra/deploy.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import urllib, json, subprocess
+
+url = 'https://registry.hub.docker.com/v2/repositories/wellcome/wellcomecollection/tags/'
+response = urllib.urlopen(url)
+data = json.loads(response.read())
+releases = [result for result in data.get('results') if result.get('name') != 'test']
+lateset_release = releases[0].get('name')
```

Comment:
nit: s/lateset/latest

Also, is it guaranteed that the Docker Hub APIs return tags in a sorted order?

Reply:
They're sorted by last added / updated.